modules/Quiz/resources/css/quiz.css

```diff
@@ -865,7 +865,7 @@ ol.second-row{
 .true-false-question .single-answer-box-bk{
     width: 41vw;
     float: left;
-    margin: 0 22px;
+    margin: 0 3vw;
```

Comment:
@MaksymKorniiov  - can we also verify this on fullscreen?

Reply:
@eitanavgil it works and looks good